modules/core/src/main/scala/org/tessellation/cli/method.scala

```diff
@@ -38,6 +38,14 @@ object method {
       environment = environment,
       http = httpConfig,
       db = dbConfig,
+      download = Download(
+        PeerDiscoveryDelay(
+          checkPeersAttemptDelay = 1.minute,
+          checkPeersMaxDelay = 10.minutes,
+          additionalDiscoveryDelay = 3.minutes,
+          minPeersCount = 2
+        )
```

Comment:
peer discovery delay should be separate from the download (as we discussed to take it off the download and make it uncoupled)

Reply:
Moved `PeerDiscoveryDelay` out of `Download`.